metadata/meson.build

```diff
@@ -3,3 +3,4 @@ install_data('autorotate-iio.xml', install_dir: wayfire.get_variable(pkgconfig:
 install_data('background-view.xml', install_dir: wayfire.get_variable(pkgconfig: 'metadatadir'))
 install_data('bench.xml', install_dir: wayfire.get_variable(pkgconfig: 'metadatadir'))
 install_data('join-views.xml', install_dir: wayfire.get_variable(pkgconfig: 'metadatadir'))
+install_data('follow-focus.xml', install_dir: wayfire.get_variable(pkgconfig: 'metadatadir'))
```

Comment:
I've been trying to keep the meson.build files containing the plugin lists in alphabetical order.